src/app/shared/components/footer/footer.component.scss

```diff
@@ -21,6 +21,7 @@
 
     .left-item {
       display: flex;
+      width: 345px;
```

Comment:
Is it possible to avoid using absolute widths?

Reply:
Changed it to percentages.